src/actions/pause/skipPausing.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+// @flow
+
+import type { ThunkArgs } from "../types";
+import { prefs } from "../../utils/prefs";
+
+/**
+ * @memberof actions/pause
+ * @static
+ */
+export function toggleSkipPausing() {
+  return async ({ dispatch, client, getState, sourceMaps }: ThunkArgs) => {
+    await dispatch({
+      type: "TOGGLE_SKIP_PAUSING"
+    });
+
+    const { skipPausing } = getState().pause;
+
+    prefs.skipPausing = skipPausing;
```

Comment:
this is a small thing, but we should set the pref in the store. We do this in reducers/ui a lot